src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -265,7 +266,11 @@ export default class DefaultDeviceController
     // Deselect any audio input devices and throw away the streams.
     // Discard the current video device, if there is one.
     // Discard any audio or video transforms.
-    await this.chooseAudioInputDevice(null);
+    const active = this.activeDevices['audio'];
```

Comment:
nit: could we put a comment explain not to call `chooseAudioInputDevice(null)` to clean up audio either here or maybe better above the `chooseAudioInputDevice` method?

Reply:
The issue is there's no API to clean up audio devices like what `chooseVideoInputDevice(null)` does. Do we need to create a method and expose it to builder? Maybe something like this:
```js
clearAudioInputDevice(){
    const active = this.activeDevices['audio'];
    if (active) {
      this.releaseActiveDevice(active);
      delete this.activeDevices['audio'];
    }
}
```

In my opinion the effect of `chooseVideoInputDevice(null)` and `chooseAudioInputDevice(null)` should be similar. Current API is quite confusing. We might want to improve this in the future major version.

Reply:
If we do not call `await this.chooseAudioInputDevice(null)` to send a fake audio stream, is it possible that the Chime backend considers the attendee dropped, e.g., in Safari?

Reply:
> If we do not call `await this.chooseAudioInputDevice(null)` to send a fake audio stream, is it possible that the Chime backend considers the attendee dropped, e.g., in Safari?

Good point, I never think of this case. Need to do some testing and confirm it.

If this is the reason why there's a difference between `chooseVideoInputDevice(null)` and `chooseAudioInputDevice(null)`, it makes a lot sense.